types/vhost/proxypass.pp

```diff
@@ -0,0 +1,16 @@
+# @summary A proxy pass definition for in an Apache vhost
```

Comment:
In the define itself there's more documentation. That probably makes sense in this file as well (or only here?)